apps/omg_watcher/test/support/watcher_helper.ex

```diff
@@ -145,6 +145,19 @@ defmodule Support.WatcherHelper do
     success?("/account.get_balance", %{"address" => Encoding.to_hex(address)})
   end
 
+  def get_blocks(page \\ 1, limit \\ 100) do
```

Comment:
defaults... please avoid them. Specify the paramters in function calls.

Reply:
wait, is this used somewhere?

Reply:
Removed